sdk/servicebus/Microsoft.Azure.ServiceBus/src/Primitives/TokenProvider.cs

```diff
@@ -74,88 +73,28 @@ public static TokenProvider CreateSharedAccessSignatureTokenProvider(string keyN
             return new SharedAccessSignatureTokenProvider(keyName, sharedAccessKey, tokenTimeToLive, tokenScope);
         }
 
-        /// <summary>Creates an Azure Active Directory token provider.</summary>
-        /// <param name="authContext">AuthenticationContext for AAD.</param>
-        /// <param name="clientCredential">The app credential.</param>
-        /// <returns>The <see cref="TokenProvider" /> for returning Json web token.</returns>
-        public static TokenProvider CreateAadTokenProvider(AuthenticationContext authContext, ClientCredential clientCredential)
-        {
-            if (authContext == null)
-            {
-                throw new ArgumentNullException(nameof(authContext));
-            }
-
-            if (clientCredential == null)
-            {
-                throw new ArgumentNullException(nameof(clientCredential));
-            }
-
-            return new AzureActiveDirectoryTokenProvider(authContext, clientCredential);
-        }
-
-        /// <summary>Creates an Azure Active Directory token provider.</summary>
-        /// <param name="authContext">AuthenticationContext for AAD.</param>
-        /// <param name="clientId">ClientId for AAD.</param>
-        /// <param name="redirectUri">The redirectUri on Client App.</param>
-        /// <param name="platformParameters">Platform parameters</param>
-        /// <param name="userIdentifier">User Identifier</param>
-        /// <returns>The <see cref="TokenProvider" /> for returning Json web token.</returns>
+        /// <param name="authCallback">The user defined authentication delegate to provide access token.</param>
+        /// <param name="authority">URL of the Azure Active Directory instance to issue token.</param>
+        /// <param name="state">Custom parameters that may be passed into the authentication delegate.</param>
+        /// <returns>The <see cref="Microsoft.ServiceBus.TokenProvider" /> for returning Json web token.</returns>
         public static TokenProvider CreateAadTokenProvider(
```

Comment:
I'd suggest we consider avoiding the abbreviation here and go with something like `CreateAzureActiveDirectoryTokenProvider` for clarity.   This class and method are `public` and, therefore, discoverable/usable by consumers directly.

Reply:
Looking at https://github.com/Azure/azure-sdk-for-net/pull/6393#issuecomment-501441965 wouldn't this change be handled in a separate (follow up) PR, @jsquire?

Reply:
Yes, but I've referred back to this one in the issue that I opened to track, since there are several comments that were not addressed due to time constraints.   I wanted to take one last pass and keep thoughts consolidated so that they're all in one place for future consideration.

Reply:
Ok. Would expect to see one comments in the new PR with a reference to this PR as a starting point. Otherwise it looks like a call to action on the PR that has been approved 🙂